docs/getting-started/local-setup/installation.md

```diff
@@ -13,7 +13,7 @@ to run on your machine.
 ## Prerequisites
 - **Git**
 - **`make` (for running Makefiles)**
-- **Go 1.19+**
```

Comment:
I'm calling this particular change out as there is a difference between the installation page and the CONTRIBUTING page. I used the higher requirement here, which is on the latter.